docs/reference/snapshot-restore/monitor-snapshot-restore.asciidoc

```diff
@@ -1,11 +1,10 @@
 [[snapshots-monitor-snapshot-restore]]
 == Monitor snapshot and restore progress
-
 ++++
 <titleabbrev>Monitor snapshot and restore</titleabbrev>
 ++++
 
-There are several ways to monitor the progress of the snapshot and restore processes while they are running. Both
+Use the <<get-snapshot-api,Get snapshot API>> or the <<get-snapshot-status-api,Get snapshot status API>> to monitor the progress of the snapshot and restore processes while they are running. Both
```

Comment:
Hmm, this is a weird sentence now but it wasn't great before either. We simply don't have a good API to monitor restores yet (I'm on it though in #59233). Maybe just remove the restore bit from this sentence for now?

Reply:
Thanks for the heads up on #59233 -- I'll keep an eye on it. For now, let's amend this sentence to read:
>Use the <get-snapshot-api,Get snapshot API>> or the <<get-snapshot-status-api,Get snapshot status API>> to monitor the progress of snapshot operations.